http_client.go

```diff
@@ -88,7 +97,11 @@ func (h *HttpClient) PostJson(uri string, jsonBytes []byte) (statusCode int) {
 		tags["statusCode"] = tags["status"]
 		log.Errorf("Unable to POST to %s: %v", uri, err)
 	} else {
-		defer resp.Body.Close()
+		defer func() {
+			if err := resp.Body.Close(); err != nil {
```

Comment:
need to check errors for `errcheck` linter, ideally we would handle this better than just logging, but meh.